boards/arm/nrf51_ble400/nrf51_ble400.dts

```diff
@@ -81,6 +81,21 @@
 	status ="ok";
 };
 
+&i2c0 {
+	status = "ok";
+	sda-pin = <0>;
+	scl-pin = <1>;
+	/* smba-pin = <2>; */
+};
+
+&spi1 {
+	status = "ok";
+	sck-pin = <25>;
+	mosi-pin = <24>;
+	miso-pin = <23>;
+	csn-pin = <30>;
```

Comment:
Huh.  The `nordic,nrf-spi.yaml` file specifies this as a `csn-pin` property and does not document that it only applies to slave mode.  Note that ble400 is one of the few boards that provide a preferred CSn pin (the 96b boards are the others I can see).

This seems to relate to #12226.

I'll have to try this to see how to make it work.

Reply:
> The `nordic,nrf-spi.yaml` file specifies this as a `csn-pin` property and does not document that it only applies to slave mode.

Yes. I think we need to improve this, and probably use separate binding for SPI slave to prevent further confusion.

Reply:
Updated.